phpunit/functional/Glpi/Helpdesk/Tile/TilesManagerTest.php

```diff
@@ -522,7 +522,7 @@ public function testFormTileWithoutNameDoesntTriggerErrors(): void
 
         // Act: render the tiles
         TemplateRenderer::getInstance()->render(
-            'pages/admin/helpdesk_home_config_tiles.html.twig',
```

Comment:
This template (and the other that have been changed below) are not specific to profiles, they are also used for entities.